db-server/index.js

```diff
@@ -100,12 +100,12 @@ function createServer(db) {
 
   api.get('/email/:email',
     op(function (req) {
-      return db.getSecondaryEmail(Buffer(req.params.email, 'hex'))
+      return db.getSecondaryEmail(req.params.email)
     })
   )
   api.get('/email/:email/account',
     op(function (req) {
-      return db.accountRecord(Buffer(req.params.email, 'hex'))
+      return db.accountRecord(req.params.email)
```

Comment:
You should be able to remove the `Buffer` from https://github.com/deeptibaghel/fxa-auth-db-mysql/blob/82073d71ad0187b55f7a3a6edf559f2345acbe8b/db-server/index.js#L113.

Reply:
Missed somehow, done now.